mne/preprocessing/maxwell.py

```diff
@@ -9,6 +9,7 @@
 from scipy.linalg import pinv
 from math import factorial
 
+from mne import pick_types
```

Comment:
all package imports should be relative, i.e. `from ..io.pick import pick_types`. If that causes a circular import you can nest the import in the function that needs it